hooks/class.tx_dlf_tcemain.php

```diff
@@ -126,7 +126,7 @@ public function processDatamap_postProcessFieldArray($status, $table, $id, &$fie
                     }
 
                     // Set port if not set.
-                    $port = (intval($conf['solrPort']) > 0 ? intval($conf['solrPort']) : 8180);
+                    $port = (intval($conf['solrPort']) > 0 ? intval($conf['solrPort']) : 8983);
```

Comment:
We should use `tx_dlf_solr::getSolrConnectionInfo()` here instead of building the base URL from configuration again.

Reply:
Should we build the base URL within the tx_dlf_solr::getSolrConnectionInfo() method or within a seperate method similar to tx_dlf_solr::getSolrUrl()?

Reply:
I just want to avoid duplicate code, so instead of reading the extension's configuration and setting the default values in `tx_dlf_tcemain->processDatamap_postProcessFieldArray()` we should use the existing method `tx_dlf_solr::getSolrConnectionInfo()`.
Building the URL from the connection info here is fine. No need to change `getSolrConnectionInfo()`!

Reply:
Done